slack/channels.py

```diff
@@ -49,6 +50,40 @@ def mark(channel, ts):
 
     return slack.http_client.post('channels.mark', data)
 
+def create(name):
+    """
+    Creates a new channel.
+    """
+    data = {
+        'token': slack.api_token,
+        'name': name,
+    }
+    return slack.http_client.post('channels.create', data)
+
+def invite(channel, user):
+    """
+    Invites a user to a channel.
+    """
+    channel_id = None
```

Comment:
I think this function should take `channel` and `user` in id format and let developers figure out how to grab id from name instead of taking care of it here.

That way we can keep this api client thin and make it independent of individual use cases.